addons/mail/static/src/discuss/call/public_web/discuss_app/client_action_patch.js

```diff
@@ -28,7 +28,7 @@ patch(DiscussClientAction.prototype, {
         if (
             hasFullScreenUrl &&
             this.store.discuss.thread?.channel.default_display_mode === "video_full_screen" &&
-            this.store.discuss.thread.rtc_session_ids.length > 0
+            this.store.discuss.thread.channel?.hasRtcSessionActive
```

Comment:
I would not introduce `hasRtcSessionActive`:
- it just add an extra getter for existing field
- name is confusing, because "active" in discuss code usually refer to "active item", i.e. `active_id` of discuss app. Here this would be best worded as `hasRtcSessions`, but I don't see much point for the getter TBH.